fixtures/varytec/giga-bar-hex-3.json

```
@@ -164,9 +172,190 @@
           "type": "NoFunction"
         },
         {
-          "dmxRange": [15, 255],
+          "dmxRange": [15, 22],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [23, 30],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [31, 40],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [40, 47],
+          "type": "ColorPreset",
+          "comment": "Pure White",
+          "colors": ["#ffffff"]
+        },
+        {
+          "dmxRange": [48, 55],
+          "type": "ColorPreset",
+          "comment": "Amber",
+          "colors": ["#ffaa00"]
+        },
+        {
+          "dmxRange": [55, 62],
+          "type": "ColorPreset",
+          "comment": "UV",
+          "colors": ["UV"]
```

```suggestion
          "colors": ["#8800ff"]
```

from https://github.com/OpenLightingProject/open-fixture-library/blob/ab74298d96f7e607c01371205d7e02793dd928ce/ui/components/global/OflSvg.vue#L60